setup.py

```diff
@@ -23,7 +26,7 @@
 with open(os.path.join(here, 'CHANGES.rst')) as f:
     CHANGES = f.read()
 
-version = '5.0.dev0'
+version = '4.3.dev0'
```

Comment:
This should be `4.4.dev0`

Reply:
Good catch, fixed in https://github.com/zopefoundation/DateTime/pull/29/commits/94e62c3485cb4a5bf9915495a1525bb2086d9737